app/templates/cli.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+'use strict';
+var chalk = require('chalk');
+var meow = require('meow');
+var <%= camelModuleName %> = require('./');
+
+var cli = meow({
+	help: [
+		'Usage',
+		'  $ <%= moduleName %> [input]',
+		'',
+		'Options',
```

Comment:
Should `Options` be above `Examles`. I've usually done `Usage`, `Exampled`, `Options`. But not sure what's optimal. ?